packages/meditrak-server/src/tests/routes/authenticate/authenticate.test.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2017 Beyond Essential Systems Pty Ltd
+ */
+
+import {} from 'dotenv/config'; // Load the environment variables into process.env
+import { expect } from 'chai';
+
+import { encryptPassword, hashAndSaltPassword } from '@tupaia/auth';
+import { findOrCreateDummyRecord, findOrCreateDummyCountryEntity } from '@tupaia/database';
+import { createBasicHeader } from '@tupaia/utils';
+
+import { TestableApp } from '../../testUtilities';
+
+const app = new TestableApp();
+const { models } = app;
+const { VERIFIED } = models.user.emailVerifiedStatuses;
+
+const userAccountPassword = 'password';
+const simpleApiClientSecret = 'simple';
+const multiPermissionApiClientSecret = 'multiPermission';
+
+let userAccount;
+let simpleApiUserAccount;
+let multiPermissionApiUserAccount;
+
+describe('Authenticate', function () {
```

Comment:
With all the permissions going on in this, it might be nice to add a summary comment or even assertion that indicates what permissions each account ends up with

Reply:
Decided to scrap a lot of the permissions stuff from these tests as we decided it might be best to not merge the accessPolicy upon return from the `/auth` endpoint.